roles/cluster_settings/README.md

```diff
@@ -0,0 +1,197 @@
+# Cluster settings role
+
+A role to define cluster settings in vCenter.
+
+## Requirements
+
+pyvmomi < 7.0.3
+
+In some cases, the vCLS cluster settings will fail to apply when using pyvmomi version 7.0.3 or greater. If this feature is required, using an earlier version will work.
+Support for version 7.0.3 and higher is planned for the next release of community.vmware (>4.4.0), at which point this restriction will be removed.
```

Comment:
So let's wait?

Reply:
this is tied to a use case we planned to include in the next release. I think its a pretty small scope bug, which is why I decided to open the pr. 

IM ok waiting, but it will be another month before community.vmware is released. Maybe we could move the affected module to vmware.vmware though?

@bardielle fyi